resources/.eslintrc-standards--annoy-mode.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    "rules": {
+        "no-magic-numbers": ["error", {                          // disallow magic numbers
+            "ignore": [0, 1, -1],
```

Comment:
I really do think we should just start to be a little more strict in our PRs when it comes to ambiguous numbers.  We do for SCSS, consider when someone adds random padding of 18px without explanation.   We tend to say 'nope, make a var or do some math based on padding'.

We should really do the same for JS.